src/execution.jl

```diff
@@ -193,6 +193,7 @@ macro roc(ex...)
                     foreach($wait!, ($(var_exprs...),))
                     if $launch
                         local $signal = $create_event($kernel.mod.exe)
+                        push!(AMDGPU.active_kernels[AMDGPU.get_default_queue()], $signal)
```

Comment:
This won't work if the queue selected by the user (and passed in `call_kwargs`) isn't the default queue.